command/v7/shared/app_summary_displayer_test.go

```diff
@@ -197,7 +249,7 @@ var _ = Describe("app summary displayer", func() {
 				It("lists instance stats for process types that have > 0 instances", func() {
 					Expect(testUI.Out).To(Say(`type:\s+web`))
 					Expect(testUI.Out).To(Say(`sidecars: `))
-					Expect(testUI.Out).To(Say(`state\s+since\s+cpu\s+memory\s+disk\s+details`))
+					Expect(testUI.Out).To(Say(`state\s+since\s+cpu\s+memory\s+disk\s+logging\s+details`))
```

Comment:
(Very) Nitpicky: Perhaps we can have a constant with the "title column" `state\s+since\s+cpu\s+memory\s+disk\s+logging\s+details` ? 
I'm not against of the current implementation. It's just an idea.